website/theme/components/Landingpage/WhoIsUsing/index.tsx

```diff
@@ -143,6 +144,12 @@ const companyList: Company[] = [
     url: 'https://verkada.com',
     width: 180,
   },
+  {
+    name: 'Miro',
+    logo: miroLogo,
+    url: 'https://miro.com',
+    width: 140,
```

Comment:
Adjust the width to 120 to keep it similar to the other logos in terms of visual size:

```suggestion
    width: 120,
```